perfmetrics/scripts/fetch_metrics.py

```diff
@@ -14,16 +15,44 @@
 FIO_WORKSHEET_NAME = 'fio_metrics'
 VM_WORKSHEET_NAME = 'vm_metrics'
 
+
+def _parse_arguments(argv):
+  """Parses the arguments provided to the script via command line.
+
+  Args:
+    argv: List of arguments recevied by the script.
+
+  Returns:
+    A class containing the parsed arguments.
+  """
+  argv = sys.argv
+  parser = argparse.ArgumentParser()
+  parser.add_argument(
+      '--upload',
+      help='Upload the results to the Google Sheet.',
+      action='store_true',
+      default=False,
+      required=False,
+  )
+  return parser.parse_args(argv[2:])
+
+
 if __name__ == '__main__':
   argv = sys.argv
-  if len(argv) != 2:
+  if len(argv) < 2:
```

Comment:
Instead we can make this fio_json_output_path a mandatory flag, and will remove this flag related check from here.

Reply:
fixed